build/patches/Experimental-user-scripts-support.patch

```diff
@@ -3096,13 +3087,12 @@ new file mode 100755
 +      return false;
 +    }
 +
++    script->set_name(user_script_path.BaseName().value());
 +    script->set_match_origin_as_fallback(MatchOriginAsFallbackBehavior::kNever);
 +
-+    // remove unicode chars and set content into File
-+    stripUnicode(content);
 +    std::unique_ptr<UserScript::File> file(new UserScript::File());
 +    file->set_content(content);
-+    file->set_url(GURL(/*script_key*/ "script.js")); // name doesn't matter
++    file->set_url(GURL(base::StrCat({"https://userscripts/file/", script->name(), ".js"})));
```

Comment:
What is this change for?

Reply:
Can you please add this to the PR description? So that I can document it in the changelog.

Can `script->name()` possibly contain dots or `/` characters?

Reply:
> Can you please add this to the PR description?

fixed.

> Can script->name() possibly contain dots or / characters?

I don't know, i'll try.

Reply:
tried, the point can be present but no problem emerges.
`/` cannot be present because between the `FilePath.BaseName()` separators used to retrieve the name